### PR TITLE
Update oe_prereqs folder install path to reflect recent changes

### PR DESCRIPTION
**Pull request by oprinmarius**
Update oe_prereqs InstallPath to reflect recent changes in our Documentation
Moved NuGet packets from C:\openenclave\prereqs\nuget. to C:\oe_prereqs

Npm installs esy by default in $env:APPDATA of current user.  This causes issue when our VHD's are built with a different user than the one being used to run the builds.

Move installation of esy to C:\oe_prereqs folder

### CODE REVIEW

**Comment by oprinmarius** [comment]
bors try

**Comment by oprinmarius** [comment]
bors try

File: .jenkins/Jenkinsfile [hunk 1]
```diff
@@ -363,8 +363,8 @@ try{
 } catch(Exception e) {
     println "Caught global pipeline exception :" + e
     GLOBAL_ERROR = e
-    throw e
```
Comment (BRMcLaren) [inline .jenkins/Jenkinsfile]: Nit: Blank space change.

**Pull request merged**